packages/modeling/package.json

```diff
@@ -4,13 +4,18 @@
   "description": "Constructive Solid Geometry (CSG) Library for JSCAD",
   "repository": "https://github.com/jscad/OpenJSCAD.org",
   "main": "src/index.js",
+  "types": "src/index.d.ts",
   "unpkg": "dist/jscad-modeling.min.js",
   "scripts": {
     "build": "browserify src/index.js -o dist/jscad-modeling.min.js -g uglifyify --standalone jscadModeling",
     "coverage": "nyc --all --reporter=html --reporter=text npm test",
     "test": "ava 'src/**/*.test.js' --verbose --timeout 2m",
+    "test:tsd": "tsd",
     "preversion": "npm run build && git add dist"
   },
+  "tsd": {
```

Comment:
seems like `tsd` only allows this configuration in the `package.json`. but [the configuration](https://github.com/SamVerschueren/tsd/#test-directory) is only to allow tests within the `src` directory, by default it would only check for an `index.test-d.ts` next to the `index.d.ts`, which seems easier to maintain if we want type tests at all.